web/Weblate-server.qmd

```diff
@@ -303,62 +303,17 @@ Weblate will provide a suggestion on how to fix, but in short:
     git svn rebase
     ```
 
-1. Optionally, if not yet done, install [helpers](https://github.com/JulienPalard/po3way) to make it easier to deal with the git conflicts (inside Docker). Start Docker as root to install `po3way` system-wide:
-
-   ```sh
-   $ sudo docker exec -u root -ti weblate-docker_weblate_1 bash
-   ```
-
-   Then in the Docker container:
-
-   ```sh
-   # pip install --prefix /usr/local po3way
-   # exit
-   ```
-
-   To set up `git` to always use `po3way`, first start the Docker container using the regular user:
-
-   ```sh
-   $ sudo docker exec -ti weblate-docker_weblate_1 bash
-   ```
-
-    And edit `.git/config` at `app/data/vcs/r-project/base-r-gui` to add the below content:
-
-    ```sh
-    [merge "po3way"]
-        name = po file merge driver
-        driver = po3way --git-merge-driver -- %A %O %B
-    ```
-
-    Also edit `.gitattributes` to include:
+1. (repeat until rebase completes) Resolve any conflicts that come up, usually using the script `deconflict_add_continue.py` found at the repo root:
```

Comment:
should we commit the Python scripts as well?

Reply:
they are committed to the Weblate SVN mirror. happy to copy them here but worry about keeping two separate copies in sync, WDYT?

Reply:
I've created https://github.com/daroczig/R-weblate-fork -- let's link the Python scripts there, and I'll set up autosyncing from weblate via GHA

Reply:
great! linked now